src/store/data_filters.ts

```diff
@@ -83,47 +83,75 @@ function contains_term(
   name: 'filteredData'
 })
 class FilteredDataModule extends VuexModule {
+  selected_file_ids: FileID[] = [];
+
+  @Mutation
+  set_toggle_file_on(file: FileID): void {
+    if (this.selected_file_ids.includes(file)) {
+      return;
+    }
+    this.selected_file_ids.push(file);
+  }
+
+  @Mutation
+  set_toggle_file_off(file: FileID): void {
+    for (let i = 0; i < this.selected_file_ids.length; i++) {
+      if (this.selected_file_ids[i] == file) {
+        this.selected_file_ids.splice(i, 1);
+      }
+    }
```

Comment:
Would be better as an `indexOf` same as above. Also the default for splice is `1` element.